website/content/en/preview/troubleshooting.md

```diff
@@ -24,6 +24,70 @@ aws ssm start-session --target $INSTANCE_ID
 # Check Kubelet logs
 sudo journalctl -u kubelet
 ```
+Here are examples of errors from Node NotReady issues that you might see from `journalctl`:
+
+* Providing the wrong block storage device name in a custom launch template can result in an error similar to:
+    ```
+    2022-01-19T18:22:23.366Z ERROR controller.provisioning Could not launch node, launching instances, with fleet error(s), InvalidBlockDeviceMapping: Invalid device name /dev/xvda; ...
```

Comment:
this is from the Karpenter logs not journalctl, and this should cause the node object to not be created at all rather than not going ready.

Reply:
I moved this message to a new section called "Node not created" and showed how to list Karpenter controller logs to view similar error conditions.